x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/advanced_detector_modal/function_help.tsx

```diff
@@ -0,0 +1,192 @@
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiButtonIcon,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiBasicTable,
+  EuiPanel,
+  EuiLink,
+  EuiText,
+} from '@elastic/eui';
+import { useMlKibana } from '../../../../../../../contexts/kibana';
+
+export const FunctionHelpPopover: FC = memo(() => {
+  const {
+    services: { docLinks },
+  } = useMlKibana();
+  const [isHelpOpen, setIsHelpOpen] = useState(false);
+
+  const onHelpClick = () => setIsHelpOpen((prevIsHelpOpen) => !prevIsHelpOpen);
+  const closeHelp = () => setIsHelpOpen(false);
+
+  const helpButton = <EuiButtonIcon onClick={onHelpClick} iconType="documentation" />;
+
+  const columnsSidebar = [
+    {
+      field: 'function',
+      name: 'Function',
+      width: '150px',
+    },
+    {
+      field: 'description',
+      name: 'Description',
+    },
+  ];
+
+  const items = [
+    {
+      id: 0,
+      function: 'count, high_count, low_count',
+      description: 'Detect anomalies when the number of events in a bucket is anomalous.',
+    },
+    {
+      id: 1,
+      function: 'non_zero_count, high_non_zero_count, low_non_zero_count',
+      description:
+        'Detect anomalies when the number of events in a bucket is anomalous, but it ignores cases where the bucket count is zero.',
+    },
+    {
+      id: 2,
+      function: 'distinct_count, high_distinct_count, low_distinct_count',
+      description: 'Detect anomalies where the number of distinct values in one field is unusual.',
+    },
+    {
+      id: 3,
+      function: 'lat_long',
+      description: 'Detects anomalies in the geographic location of the input data.',
+    },
+    {
+      id: 4,
+      function: 'info_content, high_info_content, low_info_content',
+      description:
+        'Detect anomalies in the amount of information that is contained in strings in a bucket.',
+    },
+    {
+      id: 5,
+      function: 'min',
+      description:
+        'Detects anomalies in the arithmetic minimum of a value. The minimum value is calculated for each bucket.',
+    },
+    {
+      id: 6,
+      function: 'max',
+      description:
+        'Detects anomalies in the arithmetic maximum of a value. The maximum value is calculated for each bucket.',
+    },
+    {
+      id: 7,
+      function: 'median, high_median, low_median',
+      description:
+        'Detect anomalies in the statistical median of a value. The median value is calculated for each bucket.',
+    },
+    {
+      id: 8,
+      function: 'mean, high_mean, low_mean',
+      description:
+        'Detect anomalies in the arithmetic mean of a value. The mean value is calculated for each bucket.',
+    },
+    {
+      id: 9,
+      function: 'metric',
+      description:
+        'Combines min, max, and mean functions. You can use it as a shorthand for a combined analysis. If you do not specify a function in a detector, this is the default function.',
```

Comment:
IMO The "Use it as a shorthand..." sentence could probably also be dropped.
```suggestion
        'Combine min, max, and mean functions. Use it as a shorthand for a combined analysis. This is the default function.',
```